docs: refactor client-control (#7620)
apache/apisix: docs/en/latest/plugins/client-control.md

title: client-control
keywords:
  - APISIX
  - API Gateway
  - Client Control

> description: This document contains information about the Apache APISIX client-control Plugin, you can use it to  control the behavior of Nginx to handle a client request dynamically.

Reviewer: ```suggestion
description: This document describes the Apache APISIX client-control Plugin, you can use it to control NGINX behavior to handle a client request dynamically.
```
